sp_mock/frontend/src/views/Home.vue

```diff
@@ -51,6 +49,23 @@ const logoutUser = () => {
   localStorage.removeItem("SP_TOKEN");
   router.push({ name: "loginRegister" });
 };
+
+// watch(isLoggedIn, async ()=>{
+//   let user = localStorage.getItem("_user") ? JSON.parse(localStorage.getItem("_user")) : null
+//   console.log("value: ", user)
+//   let url = await layer8_interceptor.static(user.profile_image);
+//   console.log("url: ", url)
+//   const pictureBox = document.getElementById("pictureBox");
+//   pictureBox.src = url;
+// })
+
+setTimeout(async()=>{
+  let user = localStorage.getItem("_user") ? JSON.parse(localStorage.getItem("_user")) : null
+  let url = await layer8_interceptor.static(user.profile_image);
+  const pictureBox = document.getElementById("pictureBox");
+  pictureBox.src = url;
+},500)
```

Comment:
I think the `onMounted` works, then we put this inside

Reply:
haha you caught me. "setTimeout" the universal cop out for solving async bugs. lol. 
onMounted worked. I've made the change.